### PR TITLE
Add quickcheck-state-machine proposal.

### PR DESCRIPTION
**Pull request by stevana**
Rendered: https://github.com/advancedtelematic/summer-of-haskell/blob/89d9504e90c4672b294c569902c96cdb03e4fafc/content/ideas/quickcheck-state-machine.md

### CODE REVIEW

File: content/ideas/quickcheck-state-machine.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+testing stateful/monadic code, and thereby catch up with the Erlang versions of
+QuickCheck.
+
+This proposal is about using, and possibly extending, `quickcheck-state-machine`
+in order to improve the quality of Haskell code in general and for a specific
+project in particular.
+
+The intermediate candidate could:
+
+  1) Find an interesting Haskell library or application to test;
```
Comment (jaspervdj) [inline content/ideas/quickcheck-state-machine.md]: I would add something along the lines of:
1. It should be a commonly used library
2. The library should be stateful

**Review by jaspervdj — COMMENTED** [review]
Thanks for the proposal!  I added one small comment.

File: content/ideas/quickcheck-state-machine.md (hunk 1 above)
Comment (stevana) [reply content/ideas/quickcheck-state-machine.md]: Fixed, thanks!

**Comment by jaspervdj** [comment]
@stevana Can you send me the email addresses of you and Robert?  My email address is on my github profile page.

**Comment by stevana** [comment]
Sent!

**Pull request merged**